content/events/2022-11-01-developer-week.md

```diff
@@ -5,13 +5,65 @@ ends = "2022-11-18T17:00:00"
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289623363/) | [Youtube](https://www.youtube.com/watch?v=xCxHvkS9o8g )
+
+### Developer Week: Dr. Davis & Dr. Murphy
+In the last year Hoon Education has taken-off, hundreds of people have gone through Hoon School including Dr. Murphy himself. Tune in to watch the two post-academics talk about education on Urbit, the chances for developers and what the future has in store for new Hooners.
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629398/) | [Youtube](https://www.youtube.com/watch?v=dTpw2DG6WBc)  
+
+### Developer Week: Volcano Summit
+Assembly is barely two months out and there is already a new event waiting for us. The Urbit Foundation and Tlon have teamed up with AlephDAO to organize a new developer summit. Writing runes in magma has never been so comfortable.
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629419/) | [Youtube](https://www.youtube.com/watch?v=ckFsPr-DXNE)
+
+### Developer Week: Speed Hooning with Quartus
+Nobody is quicker than Quartus, in this workshop ~rabsef-bicrym will build a reminder application from scratch that integrates with the new %homestead and show you how you can do it as well. Are you up for the challenge?
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629435/) | [Youtube](https://www.youtube.com/watch?v=eRz3s9dQZe8)
+
+### Developer Week: Core Roadmap
+Urbit Core Development is opening up to the world. Want to see where it is going? Join Philip Monk and Ted Blackman to listen to the new vision on core development, their plans for the future and what is in the pipeline already. Urbit will never be the same again.
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629450/) | [Youtube](https://www.youtube.com/watch?v=BJaRloyA02Y)
+
+### Developer Week: AMA Core Dev
+Always had a question for the core developers of Urbit, the new plans or the roadmap? Urbit’s core developers will come together in public and provide their wisdom to answer any question you might have. Join ~rovnys-ricfer, ~wicdev-wisryt, ~palfun-foslup & ~master-morzod on friday right after the Roadmap presentation.  
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629467/) | [Youtube](https://www.youtube.com/watch?v=SI_B7H8tSRE)
+
+### Developer Week: So you want to work on Urbit?
+Just finished Hoon School? Or always wanted to work for an Urbit company but don’t know where to start? In this session ~poldec-tonteg will lead a panel with several Urbit companies, about their open positions, hiring strategies and how you could become a part of it.
+
+[Meetup](https://www.meetup.com/urbit-sf/events/289629483/) | [Youtube](https://www.youtube.com/watch?v=j96Gb6dXaNs)
+
+### Developer Week: Engine Pattern
+The “trade secret” or little-known engine pattern in Hoon programs. A newer design pattern that extends working with cores to higher-level domain-specific abstractions. Join Neal Davis and Ted Blackman in their conversation about this little-known piece of magic.
```

Comment:
The first two sentences don't make sense and I'm not sure how to correct it. @sigilante ?

Reply:
Perhaps it's supposed to say this?

```suggestion
The "engine pattern" in Hoon programs is a newer design pattern that extends working with cores to higher-level domain-specific abstractions. Join Neal Davis and Ted Blackman in their conversation about this little-known piece of magic.
```